spylon_kernel/scala_magic.py

```diff
@@ -29,7 +29,7 @@ class ScalaMagic(Magic):
     def __init__(self, kernel):
         super(ScalaMagic, self).__init__(kernel)
         self.retval = None
-        self._interp = None
+        self._interp = scala_intp
```

Comment:
Hm. Is this really needed? `_get_scala_interpreter` initializes this variable by calling `get_scala_interpreter()` instead of reaching in and grabbing the global directly.